pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -11,16 +11,13 @@ isExecutable() {
 }
 
 findElfs() {
-    find "$1" -type f -exec "$SHELL" -c '
-        while [ -n "$1" ]; do
-            mimeType="$(file -b -N --mime-type "$1")"
-            if [ "$mimeType" = application/x-executable \
-              -o "$mimeType" = application/x-sharedlib ]; then
-                echo "$1"
-            fi
-            shift
-        done
-    ' -- {} +
+    find "$1" -type f | while IFS='' read -r file; do
```

Comment:
You know about `-print0`? That would be better.

Reply:
Implemented.